middleware/event_stream.go

```diff
@@ -1,27 +1,15 @@
 package middleware
 
 import (
-	"encoding/json"
-
-	"github.com/RedHatInsights/sources-api-go/internal/events"
-	"github.com/RedHatInsights/sources-api-go/kafka"
 	l "github.com/RedHatInsights/sources-api-go/logger"
-	"github.com/RedHatInsights/sources-api-go/util"
+	"github.com/RedHatInsights/sources-api-go/model"
+	"github.com/RedHatInsights/sources-api-go/service"
 	"github.com/labstack/echo/v4"
 )
 
-// producer instance used to send messages - default just an empty instance of
-// the struct.
-var producer = events.EventStreamProducer{Sender: &events.EventStreamSender{}}
-
-/*
-	Function that takes an event-type as a string and then returns a middleware
-	function that raises the specified event if the handler operation succeeds.
-
-	It bails out if there isn't a `resource` field on the context which should
-	be a model.ToEvent() call in the handler.
-*/
-func RaiseEvent(next echo.HandlerFunc) echo.HandlerFunc {
+// RaiseEventMiddleware calls the "RaiseEvent" function once the previous handler has succeeded. It grabs the resource
+// and the event type from the context.
+func RaiseEventMiddleware(next echo.HandlerFunc) echo.HandlerFunc {
```

Comment:
One last teeny thing - this can now just be `RaiseEvent` again.

Reply:
Renamed!